json_schema/examples/iec_61400-15-2_eya_def_example_a_iea43_wra_era5.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://raw.githubusercontent.com/IEA-Task-43/digital_wra_data_standard/master/schema/iea43_wra_data_model.schema.json",
+  "author": "Sergei Prokofiev",
+  "date": "2024-01-01",
+  "measurement_location": [
+    {
+      "latitude_ddeg": 53.9,
+      "longitude_ddeg": -4.3,
+      "measurement_point": [
+        {
+          "height_m": 100,
+          "height_reference_id": "ground_level",
+          "logger_measurement_config": [
+            {
+              "column_name": [
+                {
+                  "column_name": "Spd_100m_mps",
+                  "statistic_type_id": "avg"
+                }
+              ],
+              "date_from": "2000-01-01T00:00:00",
+              "date_to": "2023-12-31T23:00:00",
+              "measurement_units_id": "m/s"
+            }
+          ],
+          "measurement_type_id": "wind_speed",
+          "name": "WS100m"
+        },
+        {
+          "height_m": 100,
+          "logger_measurement_config": [
+            {
+              "column_name": [
+                {
+                  "column_name": "Dir_100m_deg",
+                  "statistic_type_id": "avg"
+                }
+              ],
+              "date_from": "2000-01-01T00:00:00",
+              "date_to": "2023-12-31T23:00:00",
+              "measurement_units_id": "deg"
+            }
+          ],
+          "measurement_type_id": "wind_direction",
+          "name": "WD100m"
+        },
+        {
+          "height_m": 2,
+          "logger_measurement_config": [
+            {
+              "column_name": [
+                {
+                  "column_name": "Tmp_2m_degC",
+                  "statistic_type_id": "avg"
+                }
+              ],
+              "date_from": "2000-01-01T00:00:00",
+              "date_to": "2023-12-31T23:00:00",
+              "measurement_units_id": "deg_C"
+            }
+          ],
+          "measurement_type_id": "air_temperature",
+          "name": "Tmp2m"
+        },
+        {
+          "height_m": 0,
+          "logger_measurement_config": [
+            {
+              "column_name": [
+                {
+                  "column_name": "Prs_0m_hPa",
+                  "statistic_type_id": "avg"
+                }
+              ],
+              "date_from": "2000-01-01T00:00:00",
+              "date_to": "2023-12-31T23:00:00",
+              "measurement_units_id": "hPa"
+            }
+          ],
+          "measurement_type_id": "air_pressure",
+          "name": "Prs0m"
+        }
+      ],
+      "measurement_station_type_id": "reanalysis",
+      "model_config": [
```

Comment:
This is an example of reference meteorological dataset metadata using the IEA Wind Task 43 WRA Data Model.